tests/test_data.py

```diff
@@ -8993,3 +9324,42 @@
 PIPE:1,14,-4
 ''',
 ]
+
+
+death_solutions = [
+r'''SOLUTION:Exploding Head Syndrome,darkid,6979-0-0,Death
```

Comment:
Remove

Reply:
I can't quite tell what you're commenting on, is this trailing whitespace?

Reply:
I cleaned up trailing whitespace, are you asking me to not check in additional tests?

There currently isn't any test coverage for DeathErrors, and it's particuarly valuable for boss levels like AMUM where the boss speed depends on player actions. This test was pretty instrumental to getting the timings right.

Reply:
No my bad, I was mixing up boss level names, thought you had left a world 4 boss test in the world 3 boss PR

Reply:
Ah, fair enough -- I added an extra test in here from the zone 5 boss just so we'd have it. In theory we should have one for all bosses, but lazy ._.